acceptance/bin/diff.py

```diff
@@ -29,6 +29,16 @@ def main():
         except re.error as e:
             print(f"Regex error for pattern {r}: {e}", file=sys.stderr)
 
+    # The acceptance testing framework replaces UUIDs with [UUID-0], [UUID-1], etc
+    # for different values of the same UUID.
+    # Since the replacements on the acceptance testing framework side are stateful,
```

Comment:
I understand the purpose of the code, but this comment is unclear:

> Since the replacements on the acceptance testing framework side are stateful

Why they are "stateful"? Perhaps it could be explained differently?